.github/workflows/publish-cli-bins.yaml

```
@@ -20,80 +20,66 @@ jobs:
+          GH_TOKEN: ${{ github.token }}
 
-      - name: Save binary
+      - name: Upload Files
         uses: actions/upload-artifact@v4
         with:
-          name: flightctl-${{ matrix.os_arch }}
-          path: flightctl-${{ matrix.os_arch }}
-
-      - name: Save checksum
-        uses: actions/upload-artifact@v4
+          name: ${{ steps.upload.outputs.artifact_name }}
```

actionlint (1.7.4)

property "upload" is not defined in object type {}

(expression)